src/server/twitterSearch.js

```diff
@@ -0,0 +1,20 @@
+var Twit = require('twit');
+
+// These keys should be hidden in a private config file or environment variables
+// For simplicity of this assignment, they will be visible here
+var T = new Twit({
+  consumer_key:         'YiSLB0kOlsTd21UGYT32YOUgg',
+  consumer_secret:      '78b5VrGzkcIkpmftLdlFwirraelPRq2t5bFlgEcMkfaQqQh1Mb',
+  access_token:         '1831536590-kX7HPRraGcbs5t9xz1wg0QdsvbOAW4pFK5L0Y68',
+  access_token_secret:  'ceYqZAulg2MT89Jw11rA44FOwHOFcEccFv9HXFIG9ckJf',
+  timeout_ms:           60 * 1000,  // optional HTTP request timeout to apply to all requests.

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'timeout_ms' is not in camel case.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2172960148/issues/source?bid=3148782&fileBranchId=3159920#l10)